draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -452,12 +454,14 @@ For Example 1, the Combined Format for Issuance looks as follows:
 For presentation to a Verifier, the Holder sends the SD-JWT and a selected
 subset of the Disclosures to the Verifier.
 
-The data format for sending the SD-JWT and the Disclosures to the Verifier is
-as follows (line break added for readability):
+The data format for sending the SD-JWT and the Disclosures to the Verifier MUST be a series of base64url-encoded values, each separated from the next by a single tilde ('~') character as follows:
 
 ```
 <SD-JWT>~<Disclosure 1>~<Disclosure 2>~...~<Disclosure M>~<optional Holder Binding JWT>
 ```
+
+The order of the base64url-encoded values MUST be an SD-JWT, Disclosures, and an optional Holder Binding JWT.
```

Comment:
```suggestion
The order of the base64url-encoded values MUST be an SD-JWT, Disclosures, and an optional Holder Binding JWT. In case there is no Holder Binding JWT, the last element is an empty string. The last separating tilde character MUST NOT be omitted.
```